recipes/plucky/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "plucky" %}
+{% set version = "0.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/plucky-{{ version }}.tar.gz
+  sha256: 5bc75d43ae6b40f1b7ba42000b37e4934fa6bd2d6a6cd4e47461f803a404c194
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python =2.6|=2.7|>=3.3
```

Comment:
Do we really want to allow python 2 in new recipes?
Only Python >=3.7 is officially supported.
I know that for noarch, it's not really an issue.

I'm just asking if there is a policy?

Reply:
Policy is that it's not needed if the author doesn't explicitly add it here. But I'd need this for an old machine which doesn't have python3 yet. 😄